BlogTemplate/Models/SlugGenerator.cs

```diff
@@ -7,12 +7,12 @@ namespace BlogTemplate.Models
 {
     public class SlugGenerator
     {
-        public string CreateSlug(string title)
+        public string CreateSlug(Post post)
         {
             BlogDataStore dataStore = new BlogDataStore();
-            string slug = title.Replace(" ", "-");
+            string slug = post.Title.Replace(" ", "-");
             int count = 0;
-            string tempSlug = slug;
+            string tempSlug = slug = $"{post.PubDate.ToFileTimeUtc()}_{slug}";
```

Comment:
Does including the timestamp in the slug allow for duplicates from different times?  E.g. 20170717_SamplePost vs. 20170718_SamplePost?